lib/determine_outcome.rb

```diff
@@ -31,9 +33,13 @@ def check_for_winner(board)
     winner
   end
 
+  def check_for_draw(board)
+   board.count { |x| x==1 || x==2}
```

Comment:
Some very tiny styling stuff. 

Currently:
```ruby
def check_for_draw(board)
 board.count { |x| x==1 || x==2}
end
```

Should be:
```ruby
def check_for_draw(board)
  board.count { |x| x == 1 || x == 2 }
end
```

Reply:
corrected this, thanks 😄